packages/frontend/src/functions/User.ts

```diff
@@ -51,6 +51,8 @@ export default class User {
           // 2. the user aborted authentication
           const status = await User.fetchUser();
           if (status === "success") {
+            // Let's auth our websocket connection as well
```

Comment:
Comment doens't provide any new information. If you believe a comment is needed, explain why it's called again